sqldelight-gradle-plugin/src/test/fixtures/well-formed-selects/expected/com/sample/HockeyPlayerModel.java

```diff
@@ -605,6 +610,64 @@ public Marshal marshal(HockeyPlayerModel copy) {
       return new Marshal(copy, birth_dateAdapter, shootsAdapter, positionAdapter);
     }
 
+    public SqlDelightStatement for_team(long _id) {
+      List<String> args = new ArrayList<String>();
+      int currentIndex = 1;
+      StringBuilder query = new StringBuilder();
+      query.append("SELECT *\n"
+              + "FROM hockey_player\n"
+              + "JOIN team ON hockey_player.team = team._id\n"
+              + "WHERE team._id = ");
+      query.append(_id);
+      query.append("");
+      return new SqlDelightStatement(query.toString(), args.toArray(new String[args.size()]));
+    }
+
+    public SqlDelightStatement question_marks_everywhere(Object expr, Object expr_, Object expr__, long expr___, Object expr____, long expr_____) {
```

Comment:
Can we do better on arg naming when unspecified? Like default to `arg1`, `arg2`, etc.